src/develop/components/StackDoc.js

```diff
@@ -0,0 +1,103 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import React, { Component } from 'react';
+import DocsArticle from '../../DocsArticle';
+import Example from '../Example';
+import Stack from '../../modules/Stack';
+
+Stack.displayName = 'Stack';
+
+export default class StackDoc extends Component {
+  render () {
+    return (
+      <DocsArticle title="Stack" colorIndex="neutral-3">
+
+        <p>A stack component.</p>
+        <pre><code className="html hljs xml">
+          {`<Stack />`}
+        </code></pre>
+
+        <section>
+          <h2>Stack Options</h2>
+          <dl>
+            <dt><code>size                 small|medium|large|xlarge</code></dt>
+            <dd>Size of content within Stack. Defaults to <code>medium</code>.</dd>
+            <dt><code>label                string</code></dt>
+            <dd>Label content.</dd>
+            <dt><code>headline             string</code></dt>
+            <dd>Headline content.</dd>
+            <dt><code>headlineStrong       true|false</code></dt>
+            <dd>If the Headline should be bold. Defaults to <code>true</code>.</dd>
+            <dt><code>paragraph            string</code></dt>
```

Comment:
The current designs only show 1 line, though it would be nice to allow multiple paragraphs. I'll let users pass either an array or a string.